README.md

```diff
@@ -49,16 +49,35 @@ or with yarn:
 yarn add  chart.js --save
 ```
 
-3. Import the `NgChartsModule` in your app main module:
+3. Import the directive in your standalone component:
 
 ```typescript
-import { NgChartsModule } from 'ng2-charts';
+import { BaseChartDirective } from 'ng2-charts';
 
-// In your App's module:
-imports: [NgChartsModule];
+@Component({
+  standalone: true,
+  imports: [BaseChartDirective],
+})
+export class MyComponent { }
 ```
 
-### Angular version compability table
+4. Provide a configuration, typically in your `main.ts`:
+
+```typescript
+import {
+  provideCharts,
+  withColorGenerator,
+  withDefaultRegisterables,
+} from 'ng2-charts';
+
+bootstrapApplication(AppComponent, {
+  providers: [
+    provideCharts(withDefaultRegisterables(), withColorGenerator()),
```

Comment:
Can you please also add a config example with a custom registerables list as you did in the docs app?

Reply:
Done in https://github.com/valor-software/ng2-charts/pull/1801/commits/0d894e54d0e70e9fd05af46f43d85e2d6cff3012